### ISSUE TITLE
Mouse buttons problem

### ISSUE THREAD

**Issue by d0ob** [FranciscoCharles/conways-game-of-life-in-python#1]
Hello there !

Here the code launches, but as soon as I click with the mouse, it tells me :
 
**File "c:\code\Python\Game_of_Life\src\main.py", line 38, in edit_automanton       
    if pygame.mouse.get_pressed(0):
ValueError: Number of buttons needs to be 3 or 5.**

From what I found online, seems to be related to number of button in pygame ? I don’t really understand…
_I have all dependencies and everything is up-to-date. And the only thing I customized in the code is assets paths._

**Comment by FranciscoCharles** [comment]
Hi d0ob, your pygame version is probably higher than **1.9.6**. I ran the installation of the newest version **2.0.1** and managed to reproduce your problem. I just uploaded the fix which also works for pygame == **2.0.1**.

giving the solution immediately, replace:

```python
    if pygame.mouse.get_pressed(0):
```
per:
```python
    if pygame.mouse.get_pressed()[0]:
```
or:
```python
    (click, *_) = pygame.mouse.get_pressed()
    if click:
```

**Comment by d0ob** [comment]
Amazing ! Works like a charm =3
Oh I see, didn’t think about that, the problem was kindof opposite so, «up-too-much-date» ^^
Evolution of Python’s «infrastructure» is something I’ll have in mind now.

_To familiarize with Python I’ve looked online multiple people’s version of the game of life, regardless of the package, the rendering, etc… Yours is incredible ! Still mostly wizardry for me ^^ But I’m deconstructing it and customizing what I can, I love it !_

**Thank you so much for the sharing ! And also for the super quick answer and fix !**
